### PR TITLE
adds "kubernetes" suffix to a directory name

### PR DESCRIPTION
**Pull request by p0lyn0mial**
**What this PR does / why we need it**: turns out that `kubetest` assumes that the last
part of that path contains "kubernetes". If not then it complains `directory: must run from kubernetes directory root`. 

See: https://github.com/kubernetes/test-infra/blob/master/kubetest/main.go#L216

**Release note**:

```
NONE
```

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
`e2e` tests for `kubeone` will use the image so that that they don't have to download additional binaries. The test use `kubetest` to run the conformance test.

File: containers/conformance-tests/install.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ apt update && apt install -y git-crypt curl
 TMP_ROOT="./.install-tmp"
 
 for VERSION in 1.10 1.11 1.12; do
-    DIRECTORY="${ROOT_DIR}/${VERSION}"
+    DIRECTORY="${ROOT_DIR}/${VERSION}-kubernetes"
```
Comment (p0lyn0mial) [inline containers/conformance-tests/install.sh:12]: will it brake already existing tests ?

**Comment by thz** [comment]
nit first:
s/"kubernets" prefix/"kubernetes" suffix/

**Comment by p0lyn0mial** [comment]
> nit first:
s/"kubernets" prefix/"kubernetes" suffix/

I've updated the title.

**Comment by alvaroaleman** [comment]
@mrIncompetent Does some magic within the `conformance-tester` to make that work, maybe you can look there? Since it works I feel like this change should not be required

**Comment by p0lyn0mial** [comment]
> @mrIncompetent Does some magic within the conformance-tester to make that work, maybe you can look there? Since it works I feel like this change should not be required

Wouldn't be better to adjust the directory to `kubetest`'s expectations ? Otherwise everyone that wants to use the image must make additional changes.

File: containers/conformance-tests/install.sh (hunk 1 above)
Comment (mrIncompetent) [reply containers/conformance-tests/install.sh:12]: Yes, it will.
We either have to adapt the conformance test tool, or we create a symlink

**Comment by mrIncompetent** [comment]
If you decide to change the image (What should be simpler than updating the code), you need to update the image tag & release the image as well. Those images are not being build & released by the CI.

See https://github.com/kubermatic/kubermatic/blob/master/containers/conformance-tests/release.sh#L3

I would recommend waiting until https://github.com/kubermatic/kubermatic/pull/2516 has been merged, as we reduced the image size there quite a bit.

**Comment by mrIncompetent** [comment]
@p0lyn0mial whats the state here?

**Comment by mrIncompetent** [comment]
Ping @p0lyn0mial

**Comment by p0lyn0mial** [comment]
sorry for being slow, do we want to change the image ? If yes, I can update places that use this img when I have time :)

**Comment by mrIncompetent** [comment]
@p0lyn0mial You opened this PR because I assume you wanted to use this image with Kubetest.
In case you don't need this anymore, please close this PR.

I remember @alvaroaleman had plans to to replace kubetest with Ginkgo ? So we might not use kubetest in the future at all

**Comment by mrIncompetent** [comment]
Closing since this seems to be not relevant anymore.
In case it becomes relevant again, reopen and consider https://github.com/kubermatic/kubermatic/pull/2521#issuecomment-448167643

**Pull request closed without merging**